### DISCUSSION TITLE
add an indication that the ingame chat screen is resizable

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#25499] [Ideas] [answered]
I've played lazer more or less exclusively for like 2 years now and I only figured this out today. Something like a handle on the top bar would work and be low enough effort. Adding variants of the cursor would also work, but that seems like it'd be more effort than it's worth
![new](https://github.com/ppy/osu/assets/49025619/725d2c57-f964-4f5b-9b59-c19143e0725d)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/25567
